### ISSUE TITLE
Azure support

### ISSUE THREAD

**Issue by exdx** [ava-labs/avalanche-ops#326]
Does avalanche-ops support building resources on Azure instead of AWS? Many customers, particularly in the financial sector, prefer to use Azure. Does it make sense to generalize the tool and support multiple clouds?

**Comment by gyuho** [comment]
Yes, this will be a great addition to the tool. And Azure has https://github.com/Azure/azure-sdk-for-rust :)

**Comment by exdx** [comment]
We may actually be more interesting in support other clouds first. Some examples include Alibaba, Google Cloud, and Oracle.
